app/jobs/api/issues_open_issue_job.rb

```diff
@@ -6,6 +6,7 @@ class IssuesOpenIssueJob < IssueEventJob
 
     def payload(params)
       {
+        pull_request: params[:pull_request],
```

Comment:
this could also be written `params[:issue].key?(:pull_request)` if the webhook payload is similar enough to the original issue payload that is.